index.js

```diff
@@ -1,7 +1,149 @@
-const getRandomNumber = (min = 0, max = 100) => {
-  const randomNumber = Math.floor(Math.random() * (max - min) + min);
+const http = require('http');
+const { db } = require('./database/database');
+const querystring = require('querystring');
+const { userNotFound, getHtmlBlock } = require('./utils/utils');
 
-  console.log(randomNumber);
-}
+const server = http.createServer((request, response) => {
+  const [url, query] = request.url.split('?');
+  const params = querystring.parse(query);
+  const { id, name, email, hobbies } = params;
 
-getRandomNumber();
+  switch (url) {
+    case '/user': {
+      const user = db.getUser(id);
+      if (!user) {
+        userNotFound(response);
+        return;
+      }
+      response.statusCode = 200;
+      response.setHeader('Content-Type', 'text/html');
+      response.end(getHtmlBlock({
+        message: 'User',
+        name: user.name,
+        email: user.email,
+      }));
+      break;
+    }
+    case '/users':
+    case '/users/': {
+      response.statusCode = 200;
+      response.setHeader('Content-Type', 'text/html');
+      const userList = db.getAllUsers();
+
+      const html = userList.reduce((rows, user) => rows + `
```

Comment:
Why didn't you reuse your `getHtmlBlock` helper here?
additionally, you could have use the combination of `map` and `join` methods